### ISSUE TITLE
Requirements,  Capabilities and Features report are not generated using cucumber5

### ISSUE THREAD

**Issue by msk-dgl** [serenity-bdd/serenity-core#2023]
serenity.version>2.2.1
serenity.maven.version>2.2.1
serenity.cucumber.version>2.2.0
cucumber.version>5.5.0

**Steps to Reproduce:**
Add tags to test scenario, for example consider the following feature file,

```
Feature: Report test feature 2

  As a user, i need to test the reports using scenario files.

  @test1
  Scenario Outline: Scenario with outline1
    Given User is testing the report
    Then print "<number>"

    @env1
    Examples:
      | number |
      | 1234   |
      | 2345   |
      | 67890  |

    @env2
    Examples:
      | number |
      | abcd   |
      | edfc   |
      | test   |

@test2
  Scenario: Scenario without outline1
    Given User is testing the report
    Then print "two"
```

**Run using** 
mvn clean verify -Dcucumber.filter.tags="@test1 and @env1" 
----- or -------
mvn clean verify -Dcucumber.options="--tags '@test1 and @env1'"

While using above commands, Test run is as expected, ie only one scenario (3 tests) is executed. But requirements, capabilities and features tab are not generated.

Sample project:
https://github.com/msk-dgl/serenity-cucumber5-starter

**Comment by wakaleo** [comment]
This was an issue in serenity-core version 2.2.0, but should be fixed in version 2.2.1. Could you  confirm that you are seeing the issue with 2.2.1 (I just checked and can't reproduce the issue with the project you mention).

**Comment by msk-dgl** [comment]
Yes, it was with 2.2.1, please run using 
mvn clean verify -Dcucumber.filter.tags="@test3 and @env1". Meanwhile I tested without outline and it works fine.

**Comment by msk-dgl** [comment]
Issue was fixed in Serenity 2.2.2. Closing the issue.
